doc/tor.1.txt

```diff
@@ -37,6 +37,25 @@ Project's website.
 
 COMMAND-LINE OPTIONS
 --------------------
+
+Tor has a powerful command-line interface. This section lists optional
+arguments you can specify at the command line using the **`tor`**
+command.
+
+NOTE: Configuration options on the command line override those in
+configuration files.  See **<<conf-format,THE CONFIGURATION FILE
+FORMAT>>** for more information.
+
+Configuration options can be specified on the command-line in the
+format **`--`**_OptionName_ _OptionValue_, or in a configuration file.
+For instance, you can tell Tor to start listening for SOCKS
+connections on port 9999 by passing either **`--SocksPort 9999`** or
+**`SocksPort 9999`** on the command line, or by specifying
+**`SocksPort 9999`** in the configuration file.  Quote option values
+that contain spaces. For instance, if you want Tor to log all
+debugging messages to **`debug.log`**, you must specify **`--Log
+"debug file debug.log"`**.
+
```

Comment:
It might be good to have a sentence here saying "The following options are only recognized on the command line"

Reply:
Which paragraph is best for that sentence, do you think? Also maybe this wording would be a little more clear? "The options described in this section are only recognized on the `tor` command line, and not in the configuration file."

Reply:
I think that reading is fine too; I think it should immediately precede the list of command-line options.